beacon-chain/core/signing/signing_root.go

```diff
@@ -247,9 +247,9 @@ func ComputeDomain(domainType [DomainByteLength]byte, forkVersion, genesisValida
 
 // This returns the bls domain given by the domain type and fork data root.
 func domain(domainType [DomainByteLength]byte, forkDataRoot []byte) []byte {
-	var b []byte
-	b = append(b, domainType[:4]...)
-	b = append(b, forkDataRoot[:28]...)
```

Comment:
I think we went down a rabbit hole at one point looking at `append` vs `copy`, but in general if we want to minimize what we're changing in a PR like this, you could default to just doing the pre-allocation while leaving the append alone. In other words I think you'd get most of the benefit by replacing `var b []byte` with `b := make([]byte, 0, 32)` but IIRC there /was/ a small benefit from copy over append but at this point so much could have changed in goroutine plus you are slicing `b` multiple times. Would have to bench to be sure.